### PR TITLE
WIP: fix(Chips): Prevent overwriting value before labels are resolved

### PR DESCRIPTION
**Pull request by trenthibbard**
When altering the value of a Chips picker, if some values have a label while others do not, the items with no label are no longer removed from the value

## **Description**

Previously, an async issue in Chips setItems() method would result in removing items from the value if there was a mix of items with labels and items without labels. This pull request will remedy this by only setting the value for the Chips element at the appropriate time

#### **Verify that...**

- [ ] Any related demos were added and `npm start` and `npm run build` still works
- [ ] New demos work in `Safari`, `Chrome` and `Firefox`
- [ ] `npm run lint` passes
- [ ] `npm test` passes and code coverage is increased
- [ ] `npm run build` still works

#### **Bullhorn Internal Developers**
- [ ] Run `Novo Automation`

##### **Screenshots**

### CODE REVIEW

File: projects/novo-elements/src/elements/chips/Chips.ts [hunk 1]
```diff
@@ -175,15 +175,13 @@ export class NovoChipsElement implements OnInit, ControlValueAccessor {
               this.items.push(value);
             }
           }
-          this._items.next(this.items);
+          this._finalizeItemValue();
```
Comment (monroepe) [inline projects/novo-elements/src/elements/chips/Chips.ts:178]: it seems like we want to call `this._finalizeItemValue();` in every scenario or close to every scenario. Could we move this call to the last line of the function (and remove the other calls to it) so it gets call every time? The logic is a little hard to follow with all the if/elses so I understand if that is not possible. I think at least in this part of the function we could move it outside of this if check.

Comment (alyssabryant) [reply projects/novo-elements/src/elements/chips/Chips.ts:178]: thanks for the suggestion! I will look at getting rid of the else blocks and just ending the method with this

Comment (alyssabryant) [reply projects/novo-elements/src/elements/chips/Chips.ts:178]: was able to simplify this a little bit, we do still need to call this new method within the `then` block for the 'noLabels' scenario in order to resolve the issue

**Review by monroepe — APPROVED** [review]

**Review by dvoegelin — APPROVED** [review]
